### ISSUE TITLE
pattern_matching_keywords generates an incorrect warning when using enums with named associated values in a switch-case 

### ISSUE THREAD

**Issue by Pispa339** [realm/SwiftLint#1819]
Version 0.22.0

Using enums with named associated values in a switch-case generates an incorrect warning.

Example:
```case .someCase(let title, _, let subtitle):```

generates a pattern_matching_keywords warning

**Comment by marcelofabri** [comment]
This is correct, you should use this instead:

```swift
case let .someCase(title, _, subtitle):
```

**Comment by Pispa339** [comment]
Seems to work. Sorry for the inconvenience and thanks for the fast reply!
